src/docs/product/data-management-settings/event-grouping/stack-trace-rules.mdx

```diff
@@ -44,9 +46,11 @@ Matches on the general platform family, which currently includes `javascript`, `
 ```discover {tabTitle:Match multiple}
 family:javascript,native stack.abs_path:**/generated/**  -group
 ```
+
```

Comment:
I think this was intentional to show multiple codeblocks as tabs

Reply:
This was done by autosave. It still renders as tabs though.